testnet/stacks-node/src/nakamoto_node/miner.rs

```diff
@@ -317,6 +318,8 @@ impl BlockMinerThread {
                         }
                     }
                 }
+                self.wait_min_time_between_blocks()?;
```

Comment:
Should `wait_min_time_between_blocks` return a bool and then we `continue` here instead of sleeping inside this function? Looking at the rest of the loop, it seems like it doesn't matter much currently, but this could be future proof in case we do something else inside this loop.

Reply:
I persomally think that is harder to follow then just the function call.

Reply:
There are lots of other similar uses of `continue` in this loop already.